slack/web/client.py

```diff
@@ -533,7 +534,7 @@ def calls_add(
                 e.g. 'https://example.com/calls/1234567890'
         """
         kwargs.update({"external_unique_id": external_unique_id, "join_url": join_url})
-        self._update_call_participants(kwargs, kwargs.get("users", None))
+        _update_call_participants(kwargs, kwargs.get("users", None))
```

Comment:
To share these methods with async client, I've extracted them from this class.

Reply:
I see, so you removed them from base client into `interal_utils`. That way when `async_client` is created, it can still reference these methods. Since async client doesn't use base client, if you left these in base client, it would have had to been copied to async base client.